dist/zone-microtask.js

```diff
@@ -1381,6 +1381,7 @@ function apply() {
       'Window',
       'Worker',
       'WorkerGlobalScope',
+      'XMLHttpRequest',
```

Comment:
Could you remove the "dist/" generated files from the commit ?
We have decided to generate them only for releases

zone & zone-microtask (.min)? .js